app/controllers/api/submission_files_controller.rb

```diff
@@ -183,7 +183,10 @@ def remove_file
       end
 
       begin
-        success, messages = grouping.remove_files([params[:filename]], @current_user)
+        success, messages = grouping.group.access_repo do |repo|
```

Comment:
Can you explain the purpose of this change? I'm not sure how this part is related to removing folders.

Reply:
when I was working on remove_folders it was using remove_files method, so I changed it here too so that it will send all the arguments to remove_files. Should I make a separate PR  for this?

Reply:
Ah okay, yes ideally this would be a separate commit and pull request, include a new test that illustrates why this change is good/necessary.